crates/re_viewer_context/src/space_view/view_query.rs

```diff
@@ -24,14 +24,22 @@ pub struct PropertyOverrides {
     /// The individual property set in this `DataResult`, if any.
     pub individual_properties: Option<EntityProperties>,
 
+    /// The recursive property set in this `DataResult`, if any.
+    pub recursive_properties: Option<EntityProperties>,
```

Comment:
At this point, I'm still confused by the meaning of "accumulated" and "recursive". Why isn't this the same thing? Also, why is `recursive_properities` and option. Shouldn't a data result always inherit from some properties?

Reply:
hmm nvm, coffe finally kicked in. Still probably worth expending a little bit in the comments, because the nomenclature is non-obvious.

Reply:
I can expand on it similar to how I did in the pull request description here. Would that make sense?

Reply:
As discussed, éet's fix the semantics first before patching up with comments :)